src/main/resources/stsjorbsmodResources/localization/spa/JorbsMod-Card-Strings.json

```diff
@@ -186,8 +186,8 @@
   },
   "stsjorbsmod:Damnation": {
     "NAME": "Perdición",
-    "DESCRIPTION": "Roba !M! cartas. NL Transforma de forma permanente la siguiente carta jugada este combate.",
-    "UPGRADE_DESCRIPTION": "Roba !M! cartas. NL Transforma de forma permanente la siguiente carta jugada este combate, luego mejórala."
+    "DESCRIPTION": "Roba !M! cartas. NL Transforma de forma permanente la próxima carta no-Legendario jugada este combate.",
+    "UPGRADE_DESCRIPTION": "Roba !M! cartas. NL Transforma de forma permanente la próxima carta no-Legendario jugada este combate, después la Mejora."
```

Comment:
I wonder if it's too confusing to include `non-Legendary` as a valid string for the `Legendary` keyword. Does the base game have cards with exclusive wordings?

Reply:
Closest thing I can think of is Pressure Points, using Mark. I don't think having "non-Legendary" as an alternative highlight text would be confusing, perhaps that's the way to go.

Reply:
Oh, there's Second Wind, Sever Soul, and Unload. Though, I guess `Attack` and `Skill` aren't keywords.